README.md

```diff
@@ -47,5 +47,7 @@ This includes all contributions from the community.
 
 ## Sponsors
 
-[![Vercel](static/assets/readme/Vercel.svg)](https://vercel.com?utm_source=game-ci)
-[![CircleCI](static/assets/readme/CircleCI.svg)](https://circleci.com?utm_source=game-ci)
+<p align="middle">
+  <a href="ttps://vercel.com?utm_source=game-ci"><img width="240" src="static/assets/readme/Vercel.svg"></a>&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp
```

Comment:
> `&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp&nbsp`

please no 😅

Reply:
yea, not the prettiest solution, but without it, the logos have no spacing between them.

Reply:
the dark background doesn't help but you can get the idea from this preview:

![image](https://user-images.githubusercontent.com/21243061/197300769-d899f50e-eee2-44b2-9b24-90ed0eafcc0b.png)

Reply:
The readme isn't a webpage. It's fine to use a table or a span with padding or whatever works.

Reply:
can we do something like

```md
<table>
  <tr>
     <td>{logo}</td>
     <td>{logo}</td>
     <td><!--empty--></td>
   </tr>
</table>
```

Or anything that's less dirty than repeating `&nbsp;` ? :)